src/api/spec/models/status_message_spec.rb

```diff
@@ -69,11 +69,11 @@
       end
 
       context 'when there is more than one announcement in her scope' do
-        let!(:status_message_for_all_users_I) { create(:status_message, severity: 'announcement', communication_scope: :all_users, created_at: 1.day.ago) }
-        let!(:status_message_for_all_users_II) { create(:status_message, severity: 'announcement', communication_scope: :all_users) }
+        let!(:status_message_for_all_users_i) { create(:status_message, severity: 'announcement', communication_scope: :all_users, created_at: 1.day.ago) }
```

Comment:
Nitpick: I would not use `i` and `ii`. Maybe renaming it to `first_status_message` and `second_status_message`

Reply:
I've applied your suggestions. Thank you!